src/workloads/query/GroupLikeDistinct.yml

```diff
@@ -77,6 +77,7 @@ Actors:
           c: {^RandomInt: {min: 0, max: 100}}
         Indexes:
           - keys: {a: 1, b: 1}
+          - keys: {b: 1}
```

Comment:
I did this instead of changing the sorter to match the original index, so that it could potentially work with the $max and $min actors too

Reply:
Changed based on slack discussion that the group `_id` needs to match the sort.